### PR TITLE
Docker refactoring

### PR DESCRIPTION
**Pull request by berezovskyi-oleksandr**

### CODE REVIEW

File: Dockerfile.tests [hunk 1]
```diff
@@ -0,0 +1,27 @@
+FROM python:3.6
+
+COPY requirements.txt /requirements.txt
+
+RUN pip download $(grep ^ansible /requirements.txt) --no-cache-dir --no-deps -d /tmp/pip && \
+    mkdir /tmp/ansible && \
+    tar -C /tmp/ansible -xf /tmp/pip/ansible* \
+    && mv /tmp/ansible/ansible* /ansible \
```
Comment (annikulin) [inline Dockerfile.tests]: Minor note: let's put all `&&` at the end on all lines for consistency.

**Comment by annikulin** [comment]
Please, update the `tox.ini` file accordingly as it still uses `test/requirements.txt` that was moved to a different folder.

File: Dockerfile.tests [hunk 2]
```diff
@@ -0,0 +1,27 @@
+FROM python:3.6
```
[the remark this answers is not in the document]
Comment (berezovskyi-oleksandr) [reply Dockerfile.tests]: Done

File: Dockerfile.tests (hunk 1 above)
Comment (berezovskyi-oleksandr) [reply Dockerfile.tests]: Done

**Comment by berezovskyi-oleksandr** [comment]
Fixed `tox.ini`

File: README.md [hunk 3]
```diff
@@ -68,18 +68,19 @@ The project contains unit tests for Ansible modules, HTTP API plugin and util fi
 ```
 docker build -t ftd-ansible-test test
 ```
-By default, the Dockerfile clones `devel` branch of Ansible repository, but you can change it by adding
-`--build-arg ANSIBLE_BRANCH=$BRANCH_NAME` to the build command.
+**NOTE**: Dockerfile uses Ansible version form `requirements.txt`. You can change it by replacing the version in `requirements.txt` and rebuilding the Docker image.
```
Comment (119Vik) [inline README.md]:
```suggestion
**NOTE**: Dockerfile uses Ansible version from `requirements.txt`. You can change it by replacing the version in `requirements.txt` and rebuilding the Docker image.
```

**Pull request merged**